src/test/scala/scala/tools/refactoring/tests/implementations/imports/OrganizeImportsTest.scala

```diff
@@ -3290,4 +3290,166 @@ class OrganizeImportsTest extends OrganizeImportsBaseTest {
     """
     }
   } applyRefactoring organizeWithTypicalParams
+
+  @Test
+  def shouldPassMatthiasBug7() = new FileSet {
```

Comment:
no worries I'm going to rename it so no hope to pass to history (without digging :) ), I just wanted these tests in suite. Actually looks that most cases are now covered (in terms of printing), I'm not sure but comments in selectors are supported too (or it is quite cheap to get it)